<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAChartAddXYSeriesWidget</class>
 <widget class="QWidget" name="DAChartAddXYSeriesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add XY Series</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxX">
         <property name="title">
          <string>X</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxX"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxXAutoincrement">
            <property name="title">
             <string>Autoincrement series</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="labelXInitValue">
               <property name="text">
                <string>Initial value</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditXInitValue">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelXStep">
               <property name="text">
                <string>Self increasing step size</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditXStepValue">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBox">
         <property name="title">
          <string>Y</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxY"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxYAutoincrement">
            <property name="title">
             <string>Autoincrement series</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="labelYInitValue">
               <property name="text">
                <string>Initial value</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditYInitValue">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelYStep">
               <property name="text">
                <string>Self increasing step size</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditYStepValue">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="DA::DACacheWindowTableView" name="tableViewXY"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DADataManagerComboBox</class>
   <extends>QComboBox</extends>
   <header>DADataManagerComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>DA::DACollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">DACollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DACacheWindowTableView</class>
   <extends>QTableView</extends>
   <header>DACacheWindowTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
